config/config.go

```diff
@@ -61,6 +63,7 @@ func init() {
 	vip.SetDefault(TradeExpiryTimeKey, 120)
 	vip.SetDefault(DataDirPathKey, defaultDataDir)
 	vip.SetDefault(PriceSlippageKey, 0.05)
+	vip.SetDefault(MacaroonRootKeyPass, "hello")
```

Comment:
I do think we may want to remove default here

Reply:
Done